deployments/deploy-ssh-gin.sh

```diff
@@ -0,0 +1,709 @@
+#!/bin/bash -x
```

Comment:
As discussed, pls create a task to clean this file up later. Also as @mkoppal-px  mentioned there shuold be a separate Dockerfile and a section in Makefile to create torpedo-api-server image. This can be taken up later but pls create a task and track it.

Reply:
Tracked under these tasks:
https://portworx.atlassian.net/browse/PB-4626
https://portworx.atlassian.net/browse/PB-4627